UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.API.FunctionalTests/FunctionalTests/ExchangeSetGenerateFilesForInValidProductVersion.cs

```diff
@@ -0,0 +1,130 @@
+﻿using Newtonsoft.Json;
+using NUnit.Framework;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+using UKHO.ExchangeSetService.API.FunctionalTests.Helper;
+using UKHO.ExchangeSetService.API.FunctionalTests.Models;
+
+namespace UKHO.ExchangeSetService.API.FunctionalTests.FunctionalTests
+{
+    class ExchangeSetGenerateFilesForInValidProductVersion
+    {
+        private string EssJwtToken { get; set; }
+        private string FssJwtToken { get; set; }
+        private ExchangeSetApiClient ExchangeSetApiClient { get; set; }
+        private FssApiClient FssApiClient { get; set; }
+        private TestConfiguration Config { get; set; }
+        public DataHelper DataHelper { get; set; }
+        private SalesCatalogueApiClient ScsApiClient { get; set; }
+        private string ScsJwtToken { get; set; }
+        private string DownloadedFolderPath { get; set; }
+        private List<ProductVersionModel> ProductVersiondata { get; set; }
```

Comment:
Rename ProductVersiondata to ProductVersionData. Do same change in ExchangeSetGenerateFilesForValidProductVersion.cs file as well

Reply:
Resolved